front/app/components/Form/Components/Controls/ImageMultichoiceControl.tsx

```diff
@@ -27,9 +25,10 @@ import { getOptions, getSubtextElement } from './controlUtils';
 import imageFile from './emptyImage.png';
 import messages from './messages';
 
-const HoverBox = styled(Box)`
+const HoverBox = styled(Box)<{ hoverColor?: string }>`
```

Comment:
```suggestion
const HoverBox = styled(Box)<{ hoverColor: string }>`
```
It's always defined in this component.